Sources/Noora/Noora.swift

```diff
@@ -302,6 +318,26 @@ public class Noora: Noorable {
         )
         try await progressStep.run()
     }
+
+    public func collapsibleStep(
+        title: TerminalText,
+        successMessage: TerminalText?,
+        errorMessage: TerminalText?,
+        visibleLines: UInt = 3,
```

Comment:
I understand we don't want to allow negative values, but I'm also not sure how I feel about using `UInt` instead of usual `Int` that most developers might expect. I'm ok with keeping the `UInt`, but I think I'd slightly lean to using `Int` even if it makes the interface a bit less type-safe.